code/modules/clothing/spacesuits/void/void.dm

```diff
@@ -200,30 +206,30 @@
 	if(H.wear_suit != src)
 		return
 
-	if(H.head == helmet)
-		to_chat(H, "<span class='notice'>You retract your suit helmet.</span>")
-		playsound(src, 'modular_citadel/sound/items/helmetdeploy.ogg', 40, 1)
-		helmet.canremove = 1
-		H.drop_from_inventory(helmet)
-		helmet.forceMove(src)
+	if(H.shoes == boots)
+		to_chat(H, "<span class='notice'>You retract your magboots.</span>")
+		boots.canremove = 1
+		H.drop_from_inventory(boots)
+		boots.forceMove(src)
 	else
-		if(H.head)
-			to_chat(H, "<span class='danger'>You cannot deploy your helmet while wearing \the [H.head].</span>")
-			return
-		if(H.equip_to_slot_if_possible(helmet, slot_head))
-			helmet.pickup(H)
-			helmet.canremove = 0
-			to_chat(H, "<span class='info'>You deploy your suit helmet, sealing you off from the world.</span>")
-			playsound(src, 'modular_citadel/sound/items/helmetdeploy.ogg', 40, 1)
-	helmet.update_light(H)
+		if(H.equip_to_slot_if_possible(boots, slot_shoes))
+			boots.pickup(H)
+			boots.canremove = 0
+			to_chat(H, "<span class='info'>You deploy your magboots.</span>")
+
+// below is code for the action button method. im dumb. but it works? if you figure out a way to make it better tell me // hey peesh i made it better -hatter
+/obj/item/clothing/suit/space/void/attack_self(mob/user)
+	toggle_helmet()
+	return
```

Comment:
Did you completely remove all the code for toggling helmets, or am I just dumb?

Reply:
^

Reply:
i just turned the code for That Shit into a proccall